pycqed/measurement/openql_experiments/openql_helpers.py

```diff
@@ -62,11 +75,20 @@ def compile(p):
     """
     Wrapper around OpenQL Program.compile() method.
     """
-    with suppress_stdout():
+    if 0:  # FIXME: allow choice
```

Comment:
This change changes the current default behaviour, the reason this with statement exists is to prevent the terminal from being spammed when using OpenQL in experiment. If I remember correctly, the log_level option does not work correctly. This may be fixed in the latest version of OpenQL.